With Flexbox/css/index.css

```diff
@@ -0,0 +1,69 @@
+body {
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1 {
+    font-family: 'Bangers', cursive;
+}
+
+a {
+    color: #b0b0b0;
+    text-decoration: none;
+}
+
+ul.flex-container {
+    display: flex;
+    justify-content: space-around;
+}
+
+.main-container {
+    margin: 2.5% auto 0;
+    width: 50%;
+}
+
+div.top {
+    margin-top: 5%;
+    display: flex;
+    justify-content: space-between;
+}
+
+.col {
+    width: 25%;
```

Comment:
I like the idea here, but this should be `width: 50%;` since the two child divs occupy 50% of the parent div's width.